tests/test_bootstrap.py

```diff
@@ -886,8 +886,10 @@ async def async_integration_setup(hass: HomeAssistant, config: ConfigType) -> bo
             dependencies=["file_upload", "http"],
         ),
     )
+
+    # We mock the MQTT integration and integrations that depend on it
```

Comment:
I'd write this instead:

```
We patch the _import platform method to avoid loading the platform module to avoid depending on non core components in the tests.
```

Reply:
Okay, clear, [I opened a PR to improve the comment](https://github.com/home-assistant/core/pull/91848)